starter-html-example/autorun.brs

```diff
@@ -0,0 +1,44 @@
+function main()
+	mp = CreateObject("roMessagePort")
+
+	' Create HTML Widget
+	widget = CreateHTMLWidget(mp)
+	widget.Show()
+
+	'Event Loop
+	while true
+		msg = wait(0,mp)
+		print "msg received - type=";type(msg)
+		if type(msg) = "roHtmlWidgetEvent" then
+			print "msg: ";msg
+		end if
+	end while
+end function
+
+function CreateHTMLWidget(mp as object) as object
+
+	' Enable Web Inspector
+	reg = CreateObject("roRegistrySection","html")
+	reg.Write("enable_web_inspector","1")
+	reg.Flush()
+
+	' Get Screen Resolution
+	vidmode = CreateObject("roVideoMode")
+	width = vidmode.GetResX()
+	height = vidmode.GetResY()
+	r = CreateObject("roRectangle",0,0,width,height)
```

Comment:
This is a thought not a legit review point but it may be worth thinking about setting the size with static numbers, purely for simplicity.

Reply:
I'm not sure what static numbers would reliably show content in full screen for multiple screen sizes. If you know of some values that reliably work, I'm happy to change. For now I will keep this dynamic range, but I like where your head is at. Thanks